### DISCUSSION TITLE
Add Daft to DuckDB Lab's H2O.ai Database-like Ops Benchmark

### DISCUSSION THREAD

**Discussion by earlev4** [Eventual-Inc/Daft#2484] [Ideas]
I want to give a huge thanks to the Eventual team and all the contributors who made the Daft project possible! I am very excited about this project.

I think it would be a great idea to add Daft to DuckDB Lab's H2O.ai Database-like Ops Benchmark. 
[https://duckdblabs.github.io/db-benchmark/](https://duckdblabs.github.io/db-benchmark/)

This benchmark currently includes other dataframe libraries such as Pandas, Dask, Modin, Polars, and more. Integrating Daft would provide valuable insights and comparisons.
For reference, here is the URL for the repository where examples could be leveraged:
https://github.com/duckdblabs/db-benchmark

Lastly, here are some articles on DuckDB Lab's H2O.ai Database-like Ops Benchmark:
[https://duckdb.org/2023/04/14/h2oai.html](https://duckdb.org/2023/04/14/h2oai.html)
[https://duckdb.org/2023/11/03/db-benchmark-update.html](https://duckdb.org/2023/11/03/db-benchmark-update.html)

Looking forward to hearing your thoughts.

Thank you!

**Comment by jaychia** [comment]
Thanks @earlev4!

We'll take a look at it, but the team is currently mostly focused on building out our new local execution engine, which will then allow us to tackle this benchmark effectively. We can take a look at it at the end of the month once we have more on that front.

**Reply by earlev4** [reply]
Thanks so much, @jaychia! I appreciate your efforts and the wonderful project!!!

**Reply by earlev4** [reply]
Hi @jaychia! I just wanted to check-in and see if there were any further considerations for adding Daft to the  DuckDB Labs' H2O.ai Database-like Ops Benchmark. Thanks!

**Reply by jaychia** [reply]
Cc @desmondcheongzx and @andrewgazelka for thoughts after the CSV reader lands

**Comment by hongbo-miao** [comment]
If adding to that benchmark, it would be awesome!
Also, found some benchmarks online (note written by Daft folks though): https://delta.io/blog/daft-delta-lake-integration/ 😃
